tools/fix_timepoint_date_problems.php

```diff
@@ -1,4 +1,4 @@
-<?php
+?php

```

Comment:
I must have been in insert mode when I thought I wasn't. I'll edit and recommit. Sorry!